javascript/eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     "no-shadow-restricted-names": 2,
     "no-undef": 2,
     "no-use-before-define": 2,
+    "quotes": [2, "double", "avoid-escape"],
```

Comment:
> What does avoid-escape do?

It looks like it allows things like `'a string containing "double quotes"'`, which personally I'm cool with.

@ABaldwinHunter my reading of [the rule docs](http://eslint.org/docs/rules/quotes) is that this will also allow ES6 backtick template literals (as long as they actually do some substitution). Can you confirm that's the case?